src/Azure.Functions.Cli/Actions/LocalActions/InitAction.cs

```diff
@@ -101,6 +105,11 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                 .WithDescription("Installs managed dependencies. Currently, only the PowerShell worker runtime supports this functionality.")
                 .Callback(f => ManagedDependencies = f);
 
+            Parser
+                .Setup<bool>("extension-bundle")
+                //.WithDescription("use default extension bundle configuration in host.json")
+                .Callback(e => ExtensionBundle = e);
```

Comment:
shouldn't this just be the default? Maybe even the only behavior?

Reply:
keep it non-default for internal testing, we need to look at the cold start numbers for this. This is why i am hiding the description.